pages/common/minidlna.md

```diff
@@ -0,0 +1,38 @@
+# minidlna
+
+> ReadyMedia (formerly MiniDLNA) is a lightweight media server compatible with DLNA/UPnP-AV clients.
+> Used to stream media to smart TVs, consoles, and other DLNA-compatible devices.
+> Configuration is typically done via the `minidlna.conf` file.
+> More information: <https://wiki.archlinux.org/title/ReadyMedia>.
```

Comment:
```suggestion
> More information: <https://manned.org/minidlna>.
```
While that is probably good documentation, I did not see any mention using it in the terminal.